doc/Language/control.pod6

```diff
@@ -912,6 +918,24 @@ for @x -> $x {
 
 prints "1245".
 
+If the C<NEXT> L<phaser|/language/phasers> is present, it runs before the next iteration:
+
```

Comment:
NEXT is also here https://docs.perl6.org/syntax/%20NEXT Maybe link for further explanation. Same goes to LAST (although there's not much to see there)

Reply:
Thanks for the pointer. If I understand correctly, it's the same as the big page on phasers, so I just linked to the section there.